_component_design/tooltip.md

```diff
@@ -287,20 +271,24 @@ Occasionally a tooltip needs to display a robust set of data that falls outside
 
 <section class="static-section" markdown="1">
 
+## Triggers
+
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Loading state
+Tooltips appear after a user activates a trigger. You can apply a trigger to an icon, table text, or data visualization.
```

Comment:
How are triggers different from the Hover and Click interactions? Does the type of trigger affect the type of interaction that makes the tooltip appear?